_examples/scalars/schema.graphql

```diff
@@ -12,10 +12,13 @@ type User {
     valPrefs: DarkMode
     ptrPrefs: DarkMode
     isBanned: Banned!
+    isLoginBanned: Boolean!
+    isQueryBanned: Boolean!
```

Comment:
As per Go spec: https://go.dev/ref/spec#Type_declarations:
- isBanned is using scalar with custom marshalling/unmarshalling
- isLoginBanned is a type definition
- isQueryBanned is an alias declaration

Making sure my PR is working fine in any of these situations.